doc/release-notes/0.24/0.24.md

```diff
@@ -0,0 +1,140 @@
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.24.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.24/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/pull/10873">#10873</a> and <a href="https://github.com/eclipse/openj9/pull/10082">#11082</a></td>
+<td valign="top">Changes to message logging</td>
+<td valign="top">All versions</td>
+<td valign="top">The <tt>-Xsyslog</tt> option replaces the <tt>-Xlog</tt> option for message logging. For compatibility with the reference implementation a limited set of <tt>-Xlog</tt> suboptions are supported.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/10175">#10175</a></td>
+<td valign="top">Support for the <strong>JAVA_OPTIONS</strong> environment variable</td>
+<td valign="top">All versions</td>
+<td valign="top">For compatibility with the reference implementation, OpenJ9 now supports the <strong>JAVA_OPTIONS</strong> environment variable for setting command line options.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/10521">#10521</a> and <a href="https://github.com/eclipse/openj9/pull/10668">#10668</a></td>
+<td valign="top"><tt>-XX:[+|-]PortableSharedCache</tt> option support update
+</td>
+<td valign="top">All versions</td>
+<td valign="top">This option is now supported on IBM Z systems&reg; and POWER&reg; systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10819">#10819</a></td>
+<td valign="top"><tt>-XX:[+|-]ShareAnonymousClasses</tt> option behavior update</td>
+<td valign="top">OpenJDK 15</td>
+<td valign="top">In earlier releases, this option controls the storage of VM anonymous classes in the shared classes cache. This option now enables and disables the storage of hidden classes in the shared classes cache.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/10774">#10774</a></td>
+<td valign="top">Additional parameters for <strong>jcmd Dump.*</strong> commands</td>
+<td valign="top">All versions</td>
+<td valign="top">Additional parameters can now be included for the <strong>jcmd Dump.*</strong> commands. For more information, see the <a href="https://www.eclipse.org/openj9/docs/version0.24/"> user documentation</a>.</td>
```

Comment:
The default options for Dump.system and Dump.heap are changed as well to use `request=exclusive+prepwalk`